README.md

```diff
@@ -45,11 +45,11 @@ This software is open source, licensed under the [Apache License 2.0][10].
 [2]: https://commons-app.github.io/
 [3]: https://github.com/commons-app/apps-android-commons/issues
 
-[4]: https://github.com/commons-app/apps-android-commons/wiki
-[5]: https://github.com/commons-app/apps-android-commons/wiki#user-documentation
-[6]: https://github.com/commons-app/apps-android-commons/wiki#contributor-documentation
-[7]: https://github.com/commons-app/apps-android-commons/wiki/Volunteers-welcome%21
-[8]: https://github.com/commons-app/apps-android-commons/wiki#developer-documentation
-[9]: https://github.com/commons-app/apps-android-commons/wiki/Libraries-used
+[4]: https://github.com/commons-app/commons-app-documentation/blob/master/android/README.md
+[5]: https://github.com/commons-app/commons-app-documentation/blob/master/android/README.md#-user-documentation
+[6]: https://github.com/commons-app/commons-app-documentation/blob/master/android/README.md#️-contributor-documentation
+[7]: https://github.com/commons-app/commons-app-documentation/blob/master/android/Volunteers-welcome!.md
```

Comment:
We could suffix this with `#volunteers-welcome` so that it leads directly to the content. This would mean the links would be: https://github.com/commons-app/commons-app-documentation/blob/master/android/Volunteers-welcome!.md#volunteers-welcome